parquet/shield-study-addon.parquetmr.txt

```diff
@@ -0,0 +1,73 @@
+message shield-study-addon {
+    required group application {
+        required binary architecture (UTF8);
+        required binary buildId (UTF8);
+        required binary channel (UTF8);
+        required binary name (UTF8);
+        required binary platformVersion (UTF8);
+        required binary version (UTF8);
+    }
+    required binary clientId (UTF8);
+    required binary creationDate (UTF8);
+    required group environment {
```

Comment:
It is only safe if you add another optional key.  In which case the parent being optional is still correct.